### ISSUE TITLE
What is Dice of the test set on the Task08_HepaticVessel dataset? Thank you!

### ISSUE THREAD

**Issue by htt1994** [Trudelutt/unet#2]

**Comment by Trudelutt** [comment]
I found that some of the images in Task08_HepaticVessel ground truth did not have the vessel structure. So I am not sure if you want that result, but if I have a result on Ircadb1 dataset.

But I would checkout my segcaps repo, if you are interested in vessel/artery segmentation

**Comment by htt1994** [comment]
Of course, I want to know the result on Ircadb1 dataset if that's all right with you. Thanks!
